apps/projects/migrations/0005_insight.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.19 on 2023-07-13 12:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("a4projects", "0038_verbose_name_created_modified"),
+        ("a4_candy_projects", "0004_verbose_name_created_modified"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Insight",
```

Comment:
@hklarner we don't need this migration anymore, the model is now called ProjectInsight.

Reply:
good point, will recreate the migrations